### ISSUE TITLE
[Bug]: Game textures & fonts become fully black to the point you can't tell if a block is front of you when using the automap feature to load map areas explored before installing JourneyMap

### ISSUE THREAD

**Issue by Chromeyyy** [TeamJM/journeymap#821]
### What happened?

I'm using the latest alpha 3 of Fabulously Optimized mod pack for Minecraft version 1.21.3 [https://modrinth.com/modpack/fabulously-optimized/version/6.4.0-alpha.3]. When I press the automap button and choose either missing or all, the game freezes for a split second and then resumes with all fonts being black. All in game textures being pitch black. You literally can't tell if a block is in front of you because all blocks are pitch black. Very weird effect. I can provide a video if you are unable to replicate this bug.

### Mod Loader

Fabric

### Minecraft Version

1.21.x

### Version

6.0.0

**Comment by mysticdrew** [comment]
provide the version of journeymap you are using
```
The binary search is a way of finding a faulty thing among a lot of other things, without having to remove the things one-by-one.
This is useful for finding a broken mod among hundreds of mods, without having to spend time testing the mods one-by-one.
The procedure is simple:
  1. Remove half of the existing things, and put them aside.
  2. Run the program / game.
  3. Does the issue still exist?
    If YES: Repeat from step 1 with the current things.
    IF NO: Swap out the current things with the ones set aside, and repeat from step 1.
  4. Repeat this process until the problematic thing/s have been found.
```

**Comment by mysticdrew** [comment]
oops didn't mean to close it

**Comment by Chromeyyy** [comment]
no worries m8. I am using the latest beta 29. Do you want me to do the binary search you said?

**Comment by mysticdrew** [comment]
> no worries m8. I am using the latest beta 29. Do you want me to do the binary search you said?

Yes, if you could please

**Comment by Chromeyyy** [comment]
so i pretty much disable half the mods and if the issue still persists, what do i do? my English isn't the best so please explain in simpler terms hehe

**Comment by mysticdrew** [comment]
Disable the remaining half.

**Comment by Chromeyyy** [comment]
I disabled 16 mods and the bug isn't happening now. Do I slowly enable each mod till I find the one that is causing it?

**Comment by Chromeyyy** [comment]
Oop, the game crashed during the automapping. so the black bug doesn't happen but crashes still occur.

**Comment by Chromeyyy** [comment]
log:
#@!@# Game crashed! Crash report saved to: #@!@# C:\Users\omark\AppData\Roaming\PrismLauncher\instances\Fabulously Optimized\minecraft\crash-reports\crash-2024-11-02_00.35.08-server.txt
Process exited with code -1.

**Comment by mysticdrew** [comment]
that is not the log, that is the path to the log

**Comment by Chromeyyy** [comment]
well im a dumbass. here you go:
[crash-2024-11-02_00.35.08-server.txt](https://github.com/user-attachments/files/17604634/crash-2024-11-02_00.35.08-server.txt)

**Comment by Chromeyyy** [comment]
I've figured out the mod which causes the black issue. It's "ferrite core version 7.1.0-hotfix" [https://modrinth.com/mod/ferrite-core/version/7.1.0-hotfix-fabric]

**Comment by Chromeyyy** [comment]
I still don't know what causes the crash since it's not instant. It happens after around 20 seconds of automap running. If you want, I can do the Binary Search again, but tomorrow as this will take more time to find as I have to wait for it to crash.

**Comment by mysticdrew** [comment]
Thanks for finding the mod, I will look into it

**Comment by Chromeyyy** [comment]
> the crash is unrelated. Looks like bad data in a chunk and when it is loaded minecraft explodes

So the a chunk in the save is corrupted pretty much? no mod is causing this so no fix for it?

**Comment by Chromeyyy** [comment]
> Thanks for finding the mod, I will look into it

Happy to help man. Thank you for providing years of support to this mod.

**Comment by Chromeyyy** [comment]
The black bug happened again with ferrite core off. Very weird. Will try to do binary search again

**Comment by Chromeyyy** [comment]
So the problem occurs on that save only. I made a new world and it doesn't happen. Save is corrupted, I guess so weird shit happens. Thanks.

**Comment by mysticdrew** [comment]
That is really strange, glad you figured it out. That is so odd it would only happen with that save
